### PR TITLE
AELIP-19 - Sponsor flow

### LINKED ISSUE #713: AELIP-19: Sponsor Flow
Update all poolCreation steps so UpfrontDeals creation is possible.

### PR DESCRIPTION
**Pull request by saeta-eth**
Closes #713

### CODE REVIEW

File: src/components/aelin/ENSOrAddress.tsx [hunk 1]
```diff
@@ -6,19 +6,21 @@ import { CellProps } from '@/src/components/pureStyledComponents/common/Table'
...
 import { ChainsValues } from '@/src/constants/chains'
 import { useEnsLookUpAddress } from '@/src/hooks/useEnsResolvers'
+import { useWeb3Connection } from '@/src/providers/web3ConnectionProvider'
 import { getExplorerUrl } from '@/src/utils/getExplorerUrl'
 import { shortenAddress } from '@/src/utils/string'
 
 interface Props extends CellProps {
   address: string
-  network: ChainsValues
+  network?: ChainsValues
```
Comment (0xlinus) [inline src/components/aelin/ENSOrAddress.tsx:15]: Why make this optional? In which scenario the network wont be defined?

**Review by 0xlinus — APPROVED** [review]

File: src/hooks/aelin/useAelinCreatePool.tsx [hunk 2]
```diff
@@ -230,7 +230,7 @@ const parseValuesToCreatePool = (createPoolState: CreatePoolStateComplete): Crea
     symbol: poolSymbol,
     purchaseTokenCap: poolCap ? parseUnits(poolCap.toString(), investmentToken?.decimals) : ZERO_BN,
     purchaseToken: investmentToken.address,
-    sponsorFee: sponsorFee ? parseEther(sponsorFee?.toString()) : ZERO_BN,
+    sponsorFee: sponsorFee ? parseUnits(sponsorFee?.toString(), 18) : ZERO_BN,
```
[the remark this answers is not in the document]
Comment (saeta-eth) [reply src/hooks/aelin/useAelinCreatePool.tsx:233]: It's the pool creation, not deal creation. Anyways I think I can create a constant for this.

File: src/components/aelin/ENSOrAddress.tsx (hunk 1 above)
Comment (saeta-eth) [reply src/components/aelin/ENSOrAddress.tsx:15]: There is a scenario where it's not defined and that's why I'm making it optional: https://github.com/AelinXYZ/aelin-frontend-v2/pull/736/files#diff-4c5b2fa69c70ae9de95adcad7e088376ab5b081ac9da6f2e03c44707ed9eb42eR245

File: src/hooks/aelin/useAelinCreatePool.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (saeta-eth) [reply src/hooks/aelin/useAelinCreatePool.tsx]: I've created a new constant, `const BASE_DECIMALS = 18`, so that we can use it when needed in the future.

**Pull request merged**